### DISCUSSION TITLE
Reduce minimum brightness?

### DISCUSSION THREAD

**Discussion by matt-sullivan** [arendst/Tasmota#12798] [Q&A] [answered]
Is it possible to reduce the minimum brightness of an RGB strip?

I'm using one as a kids night light, but even on Dimmer=1 it's still brighter than I would like. I'm using a H801 controller. I guess the pwm could go lower, but I'm not sure if/how any commands can set the lower bound. My DimmerRange is 0-100, although AFAIK that's irrelevant. I also tried MultiPwm but didn't make any real progress.

**Comment by s-hadinger** [comment, marked as answer]
If you have `LedTable 1` gamma correction, then `Dimmer 1` is the lowest possible at 1/1024 of full brightness

**Comment by JosefRypacek** [comment]
Had similar problem and solved using custom patch and modified firmware.
#18639
